pyproject.toml

```diff
@@ -22,6 +22,7 @@ atomics = "^1.0.2"
 apischema = "^0.18.0"
 xdg-base-dirs = "^6.0.0"
 typing-extensions = "^4.5.0"
+ruff = "^0.0.265"
```

Comment:
This should be added as a dev-dependency (`poetry add -D`), not as a regular dependency.